types/koa-bouncer/koa-bouncer-tests.ts

```diff
@@ -1,43 +1,35 @@
 import Koa = require("koa");
-import Router = require("koa-router");
 import bouncer = require("koa-bouncer");
 
 const app = new Koa();
 
 app.use(bouncer.middleware());
-const router = new Router()
 
-router.post('/users', async (ctx) => {
+app.use(async ctx => {
     ctx.validateBody('uname')
```

Comment:
`validateBody` was actually `any`, because this module augments `Koa.Context`, but `Router` passes in `ctx: ParameterizedContext` (`Koa.Context` extends `ParameterizedContext`).

So I took Router out of the equation here (`koa-bouncer` does not depend on it).